server/channels/app/scheduled_post.go

```diff
@@ -38,8 +40,15 @@ func (a *App) SaveScheduledPost(rctx request.CTX, scheduledPost *model.Scheduled
 		return nil, model.NewAppError("App.ScheduledPost", "app.save_scheduled_post.save.app_error", map[string]any{"user_id": scheduledPost.UserId, "channel_id": scheduledPost.ChannelId}, "", http.StatusBadRequest)
 	}
 
-	// TODO: add WebSocket event broadcast here
-
+	message := model.NewWebSocketEvent(model.WebsocketScheduledPostCreated, "", scheduledPost.ChannelId, scheduledPost.UserId, nil, connectionId)
```

Comment:
Do we need the channel ID here? Is the User ID not sufficient?

Reply:
cc. @pvev in case you missed this

Reply:
hey @esarafianou sorry, I forgot to reply. By checking the code it seems that is not needed, but I would need to double validate to be sure. I will let you know once I do this. Is just that I have my dev env in other thing now and want to finish it first before switching gears back to this.

Reply:
@pvev is there an update here? I have the update and the delete PRs pending review but we need to resolve this discussion first.

Reply:
hey @esarafianou there is no need for the channelID, I will be removing it in the delete PR, since those will be merge into this branch which contain the original changes and once those are merged, this one will be updated against master.

Reply:
@pvev Can we update the update PR as well?

Reply:
won't be necessary @esarafianou , once the delete is merged into the update, all that logic will be unified in the common function and the channelId won't be used.